WordPress/Classes/ViewRelated/Site Creation/SiteInfo/SiteInformationStep.swift

```diff
@@ -0,0 +1,18 @@
+final class SiteInformationStep: WizardStep {
+    private let creator: SiteCreator
+
+    private(set) lazy var content: UIViewController = {
+        return SiteInformationWizardContent(completion: didGoNext)
+    }()
+
+    var delegate: WizardDelegate?
+
+    init(creator: SiteCreator) {
+        self.creator = creator
+    }
+
+    private func didGoNext(_ data: SiteInformation) {
```

Comment:
The other steps include a method like `didSelect`. We're not selecting anything here, but maybe there's a common name we can come up with?

It doesn't need to block this PR, but it's something to thing about for #10458.

Reply:
Certainly. That's something I considered as well, thanks for pointing it out